bitnami/kafka/templates/controller-eligible/statefulset.yaml

```diff
@@ -0,0 +1,438 @@
+          {{- else if .Values.controller.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.controller.command "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.diagnosticMode.enabled }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
+          {{- else if .Values.controller.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.controller.args "context" $) | nindent 12 }}
+          {{- end }}
+          env:
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" (or .Values.image.debug .Values.diagnosticMode.enabled) | quote }}
+            - name: KAFKA_HEAP_OPTS
+              value: {{ coalesce .Values.controller.heapOpts .Values.heapOpts | quote }}
+            {{- if .Values.kraft.enabled }}
+            - name: KAFKA_KRAFT_CLUSTER_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{ printf "%s-kraft-cluster-id" (include "common.names.fullname" .) }}
+                  key: kraft-cluster-id
+            {{- end }}
+            {{- if and (include "kafka.saslEnabled" .) (or (regexFind "SCRAM" (upper .Values.sasl.enabledMechanisms)) (regexFind "SCRAM" (upper .Values.sasl.controllerMechanism)) (regexFind "SCRAM" (upper .Values.sasl.interBrokerMechanism))) }}
+            - name: KAFKA_KRAFT_BOOTSTRAP_SCRAM_USERS
+              value: "true"
+            {{- if and (include "kafka.client.saslEnabled" . ) .Values.sasl.client.users }}
+            - name: KAFKA_CLIENT_USERS
+              value: {{ join "," .Values.sasl.client.users | quote }}
+            - name: KAFKA_CLIENT_PASSWORDS
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "kafka.saslSecretName" . }}
+                  key: client-passwords
+            {{- end }}
+            {{- if regexFind "SASL" (upper .Values.listeners.interbroker.protocol) }}
+            - name: KAFKA_INTER_BROKER_USER
+              value: {{ .Values.sasl.interbroker.user | quote }}
+            - name: KAFKA_INTER_BROKER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "kafka.saslSecretName" . }}
+                  key: inter-broker-password
+            {{- end }}
+            {{- if and .Values.kraft.enabled (regexFind "SASL" (upper .Values.listeners.controller.protocol)) }}
+            - name: KAFKA_CONTROLLER_USER
+              value: {{ .Values.sasl.controller.user | quote }}
+            - name: KAFKA_CONTROLLER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "kafka.saslSecretName" . }}
+                  key: controller-password
+            {{- end }}
+            {{- end }}
+            {{- if .Values.metrics.jmx.enabled }}
+            - name: JMX_PORT
+              value: "5555"
```

Comment:
Any reason for hardcoding the metrics port?

Reply:
That part is imported from current logic: https://github.com/bitnami/charts/blob/e274a79d85ef6abe97bd3ebb9182fe549692829b/bitnami/kafka/templates/statefulset.yaml#L315-L318

Reply:
Yes, but now that we're performing a major bump, would it be possible to parameterize it?

Reply:
Done 👍